packages/astro/src/core/app/index.ts

```diff
@@ -52,7 +53,19 @@ export class App {
 	#base: string;
 	#baseWithoutTrailingSlash: string;
 
-	constructor(manifest: Manifest, streaming = true) {
+	async #retrievePage(routeData: RouteData) {
+		if (this.#manifest.pageMap) {
+			const pageModule = await this.#manifest.pageMap.get(routeData.component)!();
+			return await pageModule.page();
+		} else if (this.#manifest.pageModule) {
+			const pageModule = await this.#manifest.pageModule;
+			return await pageModule.page();
+		} else {
+			throw new AstroError(AstroErrorData.FailedToFindPageMapSSR);
+		}
+	}
```

Comment:
This function doesn't seem to be used, but if you do plan to keep it, private functions are often written at the bottom of the class, instead of here (before the constructor).